cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
If we are going to pass -1, then the usage text could become incorrect if the default changes from 10 consecutive.